client/state/ui/actions.js

```diff
@@ -1,7 +1,11 @@
 /**
  * Internal dependencies
  */
-import { SET_SELECTED_SITE, SET_SECTION } from 'state/action-types';
+import {
+	SET_SELECTED_SITE,
+	SET_SECTION,
+	USER_SET_CURRENT
```

Comment:
Would it be possible to update the other actions to the new naming convention in this PR or in another one closely after? Without context it'd be hard to know which naming convention is the most recent.